docs/framework_config.rst

```diff
@@ -55,23 +55,24 @@ Then, in the class's :code:`__init__` function, you need to get a config instanc
 
     class MyCog(commands.Cog):
         def __init__(self):
-            self.config = Config.get_conf(self, identifier=1234567890)
+            self.config = Config.get_conf(self, identifier=1234567890, force_registration=True)
 
 The ``identifier`` in :py:meth:`Config.get_conf` is used to keep your cog's data separate
 from that of another cog, and thus should be unique to your cog. For example: if we
 have two cogs named :code:`MyCog` and their identifier is different, each will have
-its own data without overwriting the other's data. Note that it is also possible
-to force registration of a data key before allowing you to get and set data for
-that key by adding :code:`force_registration=True` after identifier (that defaults
-to :code:`False` though)
+its own data without overwriting the other's data. Note that, as shown by most of the examples
```

Comment:
```suggestion
its own data without overwriting the other's data.

Note that, as shown by most of the examples
```
I would line break here, since it's a different topic and my other suggestion makes the `force_registration` part longer.